cs/pagination.texy

```diff
@@ -88,28 +84,22 @@ Ten zajistí, že se všechny články rozdělí do několika stránek a my zobr
 V prvním kroku si upravíme metodu pro získání článků ve třídě repositáře tak, aby nám uměla vracet jen články pro jednu stránku. Také přidáme metodu pro zjištění celkového počtu článku v databázi, kterou budeme potřebovat pro nastavení Paginatoru:
 
 /--php
-<?php
-
 namespace App\Model;
 
 use Nette;
 
-
 class ArticleRepository
 {
-	use Nette\SmartObject;
```

Comment:
Tohle může zůstat, ne?

Reply:
Spis si rikam, jestli to tam ma zustat. Potrebuje to tenhle objekt?

Reply:
No je to taková nette good practice.